azure_arc_servers_jumpstart/artifacts/Bootstrap.ps1

```diff
@@ -69,15 +69,15 @@ Start-Transcript -Path $Env:ArcBoxLogsDir\Bootstrap.log
 
 $ErrorActionPreference = 'SilentlyContinue'
 
-if ([bool]$vmAutologon) {
+#if ([bool]$vmAutologon) {

```

Comment:
Why are we removing autologon ?

Reply:
I did not see the point of having autologon as in an input parameter. Why would we need it to be selected true/false. So I removed it to keep it always autologon. 
but I can put it back if there is a good reason

Reply:
Kept autologin as default for now